### ISSUE TITLE
Why remove the aria-describedby pointing to sr-assistance when the input is filled ?

### ISSUE THREAD

**Issue by mh-nichts** [mynamesleon/aria-autocomplete#13]
The text input has its aria-describedby attribute set to point to sr-assistance at initialization, but it's removed when the input has some value (and comes back again when the input is emptied). 
We can see the logic in `setInputDescription()`.

This means that the navigation help text is not returned by screenreaders as soon as the input is filled, although it should be available at all times (especially when the user hasn't interacted yet).
Was there a problem with letting this description permanently available ?

**Comment by mynamesleon** [comment]
The general idea here was that, once a user has interacted with it and selected a value, that assistance text is then extraneous as it's implied that the user is already clear on how to use the control. I've seen screen reader testers patiently wait for everything to be read out before actually using the control that they're testing.

You're right though - the control could be populated when it initialises so users might miss this text in that scenario, and they can always just press escape, or any key, to stop the announcement.

**Comment by mh-nichts** [comment]
Thanks for the change !
